x-pack/plugins/apm/public/components/shared/charts/helper/get_alert_annotations.test.tsx

```diff
@@ -31,6 +31,7 @@ const alert: Alert = {
   tags: ['apm', 'service.name:frontend-rum'],
   'transaction.type': ['page-load'],
   'kibana.rac.alert.producer': ['apm'],
+  'kibana.rac.alert.owner': ['apm'],
```

Comment:
Nit. Wondering if we should start using the constants here. I can see us changing around naming in the next weeks as we finalize stuff, this way we won't have to rename stuff across all tests, etc.

Reply:
yeah I saw in the [pr for the new rule registry rules](https://github.com/elastic/kibana/pull/105096/files#r680145846) to start working this through. I think I will do this too 👍

Reply:
fixed (maybe..): dcea9504cf03d767eae3f8a778e2d85a759b64a5